nats-server/configuration/logging.md

```diff
@@ -81,6 +81,7 @@ For example, you could configure `logrotate` with:
 ```text
 /path/to/nats-server.log {
     daily
+    create 644 nss nss
```

Comment:
I would suggest renaming this from `nss` to something different, like or `nats`.  Longer term, the NATS streaming server is being deprecated in favor of [JetStream](https://docs.nats.io/jetstream/jetstream), so nss may not make as much sense in the future (assuming nss represents the user for nats-streaming-server).